extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -291,6 +296,17 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
 
     /** @private {number} */
     this.ifi_ = 0;
+
+    /** @private {boolean} */
+    this.isFluid_ = false;
+
+    /** @private {?string} */
+    this.fluidImpressionUrl_ = null;
+  }
+
+  /** @override */
+  isLayoutSupported(layout) {
+    return layout == Layout.FLUID || isLayoutSizeDefined(layout);
```

Comment:
Can we set this.isFluid_ here and save having to expose getLayout within src/custom-element.js?

Reply:
No can do. I need to call getLayout() in isDisplayed in resource.js (as insisted upon by two other reviewers).

Reply:
We could set `isFluid` here, though.

Reply:
Done.